specs-go/config.go

```diff
@@ -469,3 +479,77 @@ type Syscall struct {
 	Action Action `json:"action"`
 	Args   []Arg  `json:"args,omitempty"`
 }
+
+// Windows contains platform specific configuration for Windows based containers.
+type Windows struct {
+	// Resources contains constraints for containers.
+	Resources *WindowsResources `json:"resources,omitempty"`
+	// Networking contains the network connection settings for containers.
+	Networking *Networking `json:"networking,omitempty"`
+	// FirstStart is an optimization on first boot of a container.
+	FirstStart bool `json:"first_start,omitempty"`
+	// LayerFolder is the path to the current layer folder.
+	LayerFolder string `json:"layer_folder,omitempty"`
+	// Layer contains the filepaths of the parent layers.
+	LayerPaths []string `json:"layer_paths,omitempty"`
```

Comment:
These two layer settings seem like they belong to [image-spec][1] or similar.  Are they really a runtime concern?  Perhaps this will become clearer once this PR grows Markdown docs.

[1]: https://github.com/opencontainers/image-spec